src/averageAges.js

```diff
@@ -14,12 +21,15 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  return people
+    .filter(person => (person.sex === 'm')
+      && (century === undefined || century === Math.ceil(person.died / 100))
```

Comment:
in this case, you only check whether the century is explicitly undefined, but what if we have `null` there?
So it means that it is better to check not for undefined value, but to check whether it is `falsy`
```suggestion
      && (!century || century === Math.ceil(person.died / 100))
```